react/features/base/react/components/native/HeaderNavigation.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+
+import React, { Component } from 'react';
+import { View } from 'react-native';
+
+import { translate } from '../../../i18n';
+
+import BackButton from './BackButton';
+import ForwardButton from './ForwardButton';
+import HeaderLabel from './HeaderLabel';
+import styles from './headerstyles';
+
+
+type Props = {
+
+    /**
+     * Boolean to set the forward button disabled.
+     */
+    forwardDisabled: boolean,
```

Comment:
If the button is not shown when `onPressForward` is not passed, why would someone show a disabled button? And why wouldn't we allow the same for the back button?

Reply:
We don't need that possibility for the back button (there is no use-case) so I didn't implement that, trying to avoid to have a too generic component with unused functions, like the old Dialog was.

About the disabled thing: if a forward button is not rendered, then no disabled button will render. These props go together but I don't like complex props, so separated them. If there is no ```onPressForward``` then the ```forwardDisabled``` has no effect